miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditStudyController.java

```diff
@@ -77,6 +79,24 @@
   @Autowired
   private StudyService studyService;
 
+  /**
+   * Retrieves from miso.properties whether detailed sample mode has been enabled
+   * 
+   * TODO: Should this be moved further up hierarchy to avoid getting this value in every Controller that needs it?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)

Reply:
Could be moved to a different class that could be autowired into any Controllers. This would reduce the repetition of the property name (`@Value("${miso.detailed.sample.enabled}")`), which is good, but would make usage slightly more verbose.

Probably a good idea if we have multiple values from the properties file that we want to provide in this way; otherwise, I'm not sure it's worthwhile.

Either way, please complete/remove/ticket the TODO comment